apigw/v1/enums.go

```diff
@@ -5,13 +5,13 @@ import "strings"
 func EnumValuesAsString(e map[string]int32) string {
 	sb := strings.Builder{}
 	first := true
-	for k, _ := range e {
+	for k := range e {
 		if first {
 			first = false
 		} else {
-			sb.WriteString(", ")
+			_, _ = sb.WriteString(", ")
```

Comment:
Curious as to the reasoning for this change? Is it just because it better shows that the method has return values?

Reply:
It is a lint rule. There is supposed to be an exception for it, but the way it works isn't very good.